src/plots/plots.js

```diff
@@ -830,6 +830,7 @@ function applyTransforms(fullTrace, fullData, layout) {
                 fullTrace: fullTrace,
                 fullData: fullData,
                 layout: layout,
+                fullLayout: fullLayout,
```

Comment:
In fact this might allow the whole Streambed Plotly.js build to stop requiring special Axes API.